catalyst_voices/lib/configs/main_qa.dart

```diff
@@ -1,6 +1,12 @@
-import 'package:catalyst_voices/app/app.dart';
-import 'package:catalyst_voices/configs/bootstrap.dart';
+import 'package:catalyst_voices/configs/app_bloc_observer.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:go_router/go_router.dart';
+import 'package:url_strategy/url_strategy.dart';
 
-void main() async {
-  await bootstrap(() => const App());
+Future<void> main() async {
+  GoRouter.optionURLReflectsImperativeAPIs = true;
+  setPathUrlStrategy();
+  Bloc.observer = AppBlocObserver();
+
+  // await bootstrap(() => const App());
```

Comment:
I will refactor this code on your branch because it works for patrol tests but breaks the normal app launch. Most likely we need two methods, one for normal launch, another one for tests.

Reply:
I have created a new file: main_qa_patrol.dart

Reply:
Thanks, I refactored it to make sure it's synced with bootstrap